src/bindings/c/include/openvino/c/openvino.h

```diff
@@ -30,3 +30,4 @@
 #include "openvino/c/ov_remote_context.h"
 #include "openvino/c/ov_shape.h"
 #include "openvino/c/ov_tensor.h"
+#include "openvino/c/auto/auto_plugin_properties.h"
```

Comment:
```suggestion
#include "openvino/c/auto/properties.h"
```
don't use `plugin` in names